src/RadarChart.js

```diff
@@ -316,17 +248,23 @@ var radarChart = function () {
                                 .merge(verticesJoin)
                                 .attr("r", polygonPointSize)
                                 .attr("cx", function(d, i) {
-                                    // console.log(d.coordinates.x);
                                     return d.coordinates.x;
                                 })
                                 .attr("cy", function(d, i) {
-                                    // console.log(d.coordinates.y);
                                     return d.coordinates.y;
                                 })
                                 .attr("fill", color(gColor))
                                 .attr("fill-opacity", 1)
-                                .on(over, showTooltip)
-                                .on(out, verticesTooltipHide);
+                                .on(over, function(d) {
+                                    verticesTooltip.style("opacity", 0.9)
```

Comment:
I'm slightly concerned that doing this with opacity will leave a "ghost" element covering something up. Does it work if you do `visibility: hidden` and `visibility: visible` instead? That's what we use for the tooltips on other charts.